File: app/src/main/java/com/marchelo/developerslite/model/PostsListHolder.java
package com.marchelo.developerslite.model;

import java.util.ArrayList;
import java.util.List;

import com.google.gson.annotations.Expose;

/**
 * @author Oleg Green
 * @since 23.08.15
 */
public class PostsListHolder {
    @Expose private List<Post> result = new ArrayList<>();
    @Expose private Integer totalCount;

    public PostsListHolder(List<Post> posts) {
        result = posts;
    }
    /**
     * @return The result
     */
    public List<Post> getList() {
        return result;
    }

    /**
     * @return The totalCount
     */
    public Integer getTotalCount() {
        return totalCount;
    }
}

File: app/src/main/java/com/marchelo/developerslite/model/PostsListHolder.kt
package com.marchelo.developerslite.model

import java.util.ArrayList

import com.google.gson.annotations.Expose

/**
 * @author Oleg Green
 * *
 * @since 23.08.15
 */
class PostsListHolder(posts: List<Post>) {
    /**
     * @return The result
     */
    @Expose
    val list: List<Post> = posts
    /**
     * @return The totalCount
     */
    @Expose
    val totalCount: Int? = posts.count();
}